README.md

```diff
@@ -14,8 +14,13 @@ download or the class path that [JDTLS] uses in your Zed settings like so:
   "lsp": {
     "jdtls": {
       "settings": {
-        "version": "1.40.0", // This is the default value
-        "classpath": "/path/to/classes.jar:/path/to/more/classes/"
+        "classpath": "/path/to/classes.jar:/path/to/more/classes/",
+        "jdtls": {
+          "version": "1.40.0", // This is the default value
+        },
+        "lombok": { // Only needed if `initialization_options.settings.java.jdt.ls.lombokSupport.enabled` is set to `true`
+          "version": "1.18.34", // Defaults to the latest version if not set
+        }
```

Comment:
I'm not sure I like this, there's not really a reason to have the `version` fields be nested since there are no other fields to be grouped together. It *would* take more time for someone to type this out as opposed to just `jdtls_version` or `lombok_version`.

Reply:
Changed to `settings.(jdtls|lombok)_version`.